src/SDKs/GuestConfiguration/Management.GuestConfiguration/Microsoft.Azure.Management.GuestConfiguration.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
```

Comment:
Please remove the .nupkg files checked in

Reply:
removed